config/settings/gateway.q

```diff
@@ -9,6 +9,9 @@ querykeeptime:0D00:30		// the time to keep queries in the
 errorprefix:"error: "		// the prefix for clients to look for in error strings
 clearinactivetime:0D01:00	// the time to keep inactive handle data
 
+// list of process types connected for sync reload
+connectedProcs:()
```

Comment:
we don't use camel case anywhere else in TorQ, so let's not introduce it here please